boards/arm/nrf52833_pca10100/nrf52833_pca10100.dts

```diff
@@ -194,9 +194,9 @@ arduino_i2c: &i2c0 {
 
 arduino_spi: &spi3 {
 	status = "okay";
-	sck-pin = <41>;
-	miso-pin = <43>;
-	mosi-pin = <42>;
+	sck-pin = <23>;
+	miso-pin = <22>;
+	mosi-pin = <21>;
```

Comment:
Pin 21 is currently assigned as MISO line in `spi2`.
I propose to remove the `spi2` node completely.

Reply:
@anangl What is about i2c1, spi0 and spi1 nodes, what is the reason to have this?

Reply:
Consistency with nrf52840_pca10056, the nodes use the same pins on both boards.
`spi3` has to use different pins, obviously.
`spi2` in nrf52840_pca10056 serves for communication with the NOR flash, which is not present on nrf52833_pca10100, hence I proposed to remove the pin assignments for it, instead of trying to find some unoccupied pins.